src/components/MainContent/index.jsx

```diff
@@ -23,8 +23,8 @@ const MainContent = (props) => {
               {/* TODO: Change to slugs not ids */}
               <Route exact path="/" component={IndexPage} />
               <Route path="/:journalId/about" component={JournalAboutPageContainer} />
-              {/* <Route path="/:journalId" component={JournalRerouter} /> */}
               <Route path="/:journalId/pages/:pageId" component={JournalPage} />
+              {/* <Route path="/:journalId" component={JournalRerouter} /> */}
```

Comment:
is JournalRerouter going to come in a separate PR? I assume that is what we discussed yesterday, routing to either the first page or last page visited

Reply:
Yeah, I'm working on that now. I arbitrarily picked the PR cutoff before it I implemented that.